src/uvw/stream.hpp

```diff
@@ -155,8 +155,8 @@ class StreamHandle: public Handle<T, U> {
 
 public:
 #ifdef _WIN32
```

Comment:
@tusharpm 
OP says that changing it to `#ifdef _MSC_VER` works fine for MinGW, but it breaks out MSVC.  
I've not a Win env to test it, set aside appveyor. Do you have any chance to look over the change and find what's wrong?  
Let me know. Thank you very much in any case.